nni/retiarii/evaluator/pytorch/cgo/trainer.py

```diff
@@ -2,7 +2,13 @@
 # Licensed under the MIT license.
 
 import pytorch_lightning as pl
-from .accelerator import BypassAccelerator
+from pytorch_lightning.strategies import SingleDeviceStrategy
```

Comment:
I suppose the pyright check could be easier after deleting one file. Could you try to remove that from ignore?